create-neon/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "create-neon",
+  "version": "0.1.0",
+  "description": "Create Neon projects with no build configuration.",
+  "author": "Dave Herman <david.herman@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/neon-bindings/neon/issues"
+  },
+  "homepage": "https://github.com/neon-bindings/neon#readme",
+  "bin": "./dist/bin/create-neon.js",
+  "files": [
+    "dist/**/*",
+    "templates/**/*"
```

Comment:
Nit, it would be nice to precompile the templates. It's not necessary because performance of this package isn't critical.